### PR TITLE
Fixed - IDENTITY-4207

### PR DESCRIPTION
**Pull request by JKAUSHALYA**
https://wso2.org/jira/browse/IDENTITY-4207

### CODE REVIEW

File: components/user-mgt/org.wso2.carbon.identity.user.registration/src/main/java/org/wso2/carbon/identity/user/registration/UserRegistrationException.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
+ *
```
Comment (omindu) [inline components/user-mgt/org.wso2.carbon.identity.user.registration/src/main/java/org/wso2/carbon/identity/user/registration/UserRegistrationException.java]: Unnecessary new line ?

Comment (JKAUSHALYA) [inline components/user-mgt/org.wso2.carbon.identity.user.registration/src/main/java/org/wso2/carbon/identity/user/registration/UserRegistrationException.java]: Fixed in https://github.com/wso2/carbon-identity/pull/1690

File: components/user-mgt/org.wso2.carbon.identity.user.registration/src/main/java/org/wso2/carbon/identity/user/registration/UserRegistrationService.java [hunk 2]
```diff
@@ -201,13 +201,14 @@ public boolean isAddUserWithInfoCardEnabled() throws Exception {
      * @return True if exist.
      * @throws Exception
      */
-    public boolean isUserExist(String username) throws Exception {
+    public boolean isUserExist(String username) throws UserRegistrationException {
```
Comment (sinthuja) [inline components/user-mgt/org.wso2.carbon.identity.user.registration/src/main/java/org/wso2/carbon/identity/user/registration/UserRegistrationService.java]: Remove unnecessary blank lines throughout the code.

Comment (JKAUSHALYA) [inline components/user-mgt/org.wso2.carbon.identity.user.registration/src/main/java/org/wso2/carbon/identity/user/registration/UserRegistrationService.java]: Fixed in https://github.com/wso2/carbon-identity/pull/1701

**Pull request merged**